src/components/quote.jsx

```diff
@@ -0,0 +1,24 @@
+async function emoji(callback) {
+  const name = 'slightly%20smiling%20face';
+  const apiKey = '2VyJFEAwAfCwJeQRTJpUKw==qoL1KftFFfAoQ4th';
+
+  try {
```

Comment:
- [ ] Please ensure that you create a `<Quotes>` component and in this component make sure the `useEffect` hook has been used correctly
    1. There should be a `loading` state for the API call
    2. There should be an `error` state for the API call.